src/Components/Publishing/Display/Canvas/CanvasContainer.tsx

```diff
@@ -75,7 +75,7 @@ export class CanvasContainerComponent extends React.Component<CanvasContainerPro
 
   render() {
     const { campaign, disclaimer, size, unit } = this.props
-    const { assets, layout, link: { url } } = unit
+    const { assets, cover_image_url, layout, link: { url } } = unit
```

Comment:
Don't we need to pull this in from the panel?

Reply:
Ahh I see: `this.props.display.canvas` is what `unit` resolves to further up the chain

Reply:
Nevermind, discussed irl!

Reply:
As we talked about offline, gonna patch this at the force query level